ratis-common/src/test/java/org/apache/ratis/BaseTest.java

```diff
@@ -85,8 +83,6 @@ public void setup(TestInfo testInfo) {
         + "." + (method == null? null : method.getName());
   }
 
-  // @Before annotation is retained to support junit 4 tests.
```

Comment:
Let's also remove the other two junit 4 comments:
```diff
+++ b/ratis-common/src/test/java/org/apache/ratis/BaseTest.java
@@ -72,8 +72,6 @@ public abstract class BaseTest {
     }
   }
 
-  // TODO: Junit 4 reference should be removed once all the unit tests are migrated to Junit 5.
-
   private String testCaseName;
 
   @BeforeEach
@@ -133,7 +131,6 @@ public abstract class BaseTest {
   }
 
   public File getTestDir() {
-    // This will work for both junit 4 and 5.
     return new File(getClassTestDir(), testCaseName);
   }
 
```

Reply:
Thank you for pointing out these issues! I have made the changes as suggested.